tests/python/pants_test/backend/python/test_python_chroot.py

```diff
@@ -82,6 +82,38 @@ def dumped_chroot(self, targets):
       finally:
         python_chroot.delete()
 
+  def test_shebang_modified(self):
```

Comment:
These tests should be reverted now that you have `test_shebang_modified` in `tests/python/pants_test/backend/python/tasks2/test_python_binary_create.py`.